news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "\"Medal Hunter\" Title's Second Coming"
+date: 2023-08-28 03:00:00 +0000
+---
+
+The legendary "**Medal Hunter**" title came back once again challenging every player to find a solution for a new medal!
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/banner.jpg)
+
+[Medals](/wiki/Medals) are osu! achievements which are given for achieving certain goals, and the last batch of medals was quite exciting.
+On July 27th, 2023, alongside new [Hush-Hush medals](/wiki/Medals#hush-hush), [Ephemeral](https://osu.ppy.sh/users/102335) made a [news post](https://osu.ppy.sh/home/news/2023-07-27-summer-splash) in which he announced the "**Medal Hunter**" title coming back once again.
+
+"**Medal Hunter**" is a [unique user title](/wiki/People/Users_with_unique_titles), which is given for being the first player to unlock an extremely hard to solve Hush-Hush medal. The [previous "**Medal Hunter**" title](https://osu.ppy.sh/home/news/2022-09-09-how-zetera-became-a-medal-hunter) has been given out nearly **6 years ago**, which definitely made it even more interesting.
+
+In order to obtain such a desirable title, you had to be the first player to find the solution for the **Internment** medal and unlock it. **Internment** is a Hush-Hush medal added in the last batch, being so hint-based that developers found unlocking it worth a title. Speaking of hint and sub-hint (referenced as flairs by Ephemeral), they are "Stained in the hues of madness, they rage within a four-walled jail." and "Bask in the shade of an ailing mind." respectively.
+And while only one player could get a shiny "**Medal Hunter**" text under their username, many still have decided to team up with others on the [Medal Hunters Discord server](https://discord.gg/8qpNTs6)!
+
+But before we get to community's attempts to solve it, here are some details they knew from the beginning: 
+The medal is only visible in osu!standard profile, so it's std-only, meaning you can't get it in other gamemodes; 
+If the medal is map-specific, meaning it can only be done on specific mapset(s) - it has to be Featured Artist song; 
+The medal is in a Hush-Hush (Expert) category, which means no [difficulty reduction mods](/wiki/Gameplay/Game_modifier#difficulty-reduction) (or [special mods](/wiki/Gameplay/Game_modifier#special)) are allowed.
```

Comment:
should be bullet points not separate rows

Reply:
will do